src/app/Http/ViewHelpers/Employee/EmployeeLogViewHelper.php

```diff
@@ -50,22 +49,4 @@ public static function employee(Employee $employee): array
             'avatar' => ImageHelper::getAvatar($employee, 80),
         ];
     }
-
-    /**
-     * Collection containing audit log types for this employee.
-     * @param mixed $logs
-     */
-    public static function employeeLogTypes($logTypes): Collection
```

Comment:
This function to transform the collection isn't needed, Vue is able to handle the collection that is returned already